assets/stylesheets/screens/_result.scss

```
@@ -2,4 +2,24 @@
   .card-upload {
     @apply sm:max-w-lg;
   }
+
+  .title-feature {
+    @apply mt-8 mb-4 text-2xl text-center leading-8 font-bold tracking-tight text-primary-900 sm:text-4xl;
+  }
+
+  .table-wrapper {
+    @apply flex flex-wrap justify-around;
+  }
+
+  .table-results {
```

We can use singular class name as per our convention
```suggestion
  .table-result {
```

---

✅  fixed in [4c10b49](https://github.com/carryall/go-google-scraper-challenge/pull/66/commits/4c10b490b9cf075bde20e98c4da06b0592c387a3)